flake.nix

```diff
@@ -82,6 +82,11 @@
           description = "Build a workspace with a trunk member";
           path = ./examples/trunk-workspace;
         };
+
+        end-to-end-testing = {
```

Comment:
Nit: These are generally sorted alphabetically if you don't mind moving this up a tad!

Reply:
Done. I let `default` stay together with `quick-start` though, as formatting seems to imply `default` is not an actual item on the list.